### ISSUE TITLE
Highlighting other languages, such as "shell session" in tutorials

### ISSUE THREAD

**Issue by panglesd** [ocaml/ocaml.org#559]
Currently, if I understand well, only ocaml-related syntax is highlighted, such as interface/implementation ocaml file, opam file and dune file.

However, highlighting for "shell session" could be nice to have, for example in the tutorials on setting the ocaml environment up, where there are several commands to run displayed.

Using highlight-js could render as follows (theme chosen at random):
![image](https://user-images.githubusercontent.com/34110029/186441470-849ac249-14b7-417a-af19-50f8aefed4b5.png)

**Comment by patricoferris** [comment]
https://github.com/patricoferris/hilite/pull/4 adds basic support inn hilite for bash/shell -- works for comments but most things just get shell source. Some examples:

<img width="831" alt="Screenshot 2022-09-02 at 17 31 56" src="https://user-images.githubusercontent.com/20166594/188198537-8ad40ff9-ae41-40e3-8ddc-e1edfd0c595d.png">

<img width="842" alt="Screenshot 2022-09-02 at 17 32 29" src="https://user-images.githubusercontent.com/20166594/188198615-75c9d27e-238e-47e6-bf20-c575722bc76b.png">

**Comment by cuihtlauac** [comment]
Pending on availability of hilite 0.1.1. 
https://github.com/ocaml/opam-repository/pull/22551

**Comment by tmattio** [comment]
This is fixed, right @cuihtlauac?
